manim/animation/creation.py

```diff
@@ -563,15 +563,22 @@ def __init__(
         **kwargs,
     ) -> None:
         self.time_per_char = time_per_char
+
+        # Check for empty text using family_members_with_points()
+        if not text.family_members_with_points():
+            raise ValueError(
+                "The input text is empty i.e., input has no characters to read thus cannot render a video file."
+            )
+
         if run_time is None:
-            # minimum time per character is 1/frame_rate, otherwise
-            # the animation does not finish.
             run_time = np.max((1 / config.frame_rate, self.time_per_char)) * len(text)
+
         super().__init__(
             text,
             suspend_mobject_updating=suspend_mobject_updating,
             int_func=int_func,
             rate_func=rate_func,
+            time_per_char=time_per_char,
```

Comment:
The parent class, `ShowIncreasingSubsets` does not seem to have a `time_per_char` keyword argument -- this should not be propagated upwards.

Reply:
Understood got it!